packages/connection/src/class-plugin-storage.php

```diff
@@ -19,6 +19,22 @@
  */
 class Plugin_Storage {
 
+	const ACTIVE_PLUGINS_OPTION_NAME = 'connection_plugins_active';
```

Comment:
`ACTIVE_PLUGINS` looks kind of ambiguous, may be `CONNECTED_PLUGINS` would be more self-explanatory?